python-package/lightgbm/dask.py

```diff
@@ -239,7 +239,7 @@ def _predict_part(part, model, proba, **kwargs):
         result = model.predict(data, **kwargs)
 
     if isinstance(part, pd.DataFrame):
-        if proba:
+        if proba or kwargs.get('pred_contrib', False):
```

Comment:
I'd better rename `proba` to `pred_proba` and make other available in sklearn wrapper `pred_*` params ordinary function arguments to unify interfaces.

Reply:
totally agree! I initially didn't do that because I was trying to keep the diffs small. But I'll do that on this PR.

Reply:
ok I tried this in https://github.com/microsoft/LightGBM/pull/3774/commits/98a787860dbe4e10879e76cd79c9ad4a55e44d03, is that what you meant?

Reply:
Yeah, thanks! Exactly what I meant. Love it!